### PR TITLE
WIP experiment with firehose

### PR DESCRIPTION
**Pull request by mohit**
Conflicts:
    package.json
    pages/main-parent/actions/index.jsx
    pages/main-parent/components/HomePage.jsx
    pages/main-parent/components/add-student/AddStudentPage.jsx
    pages/main-parent/routes.jsx

### CODE REVIEW

File: pages/main-parent/components/HomePage.jsx [hunk 1]
```diff
@@ -37,7 +49,8 @@ function mapStateToProps(state) {
 // currently not used for anything, no actions triggered on this page
 function mapDispatchToProps (dispatch) {
   return {
-    getStudentList: function(){ dispatch(actions.getStudentList()); }
+    getStudentList: function(){ dispatch(actions.getStudentList()); },
+    getPilotsForDistrict: function(){ dispatch(actions.getPilotsForDistrict()); },
```
Comment (ipam73) [inline pages/main-parent/components/HomePage.jsx]: probably want to remove this line :)

**Comment by ipam73** [comment]
feel free to merge if it works :) we can work out the hacky-ness later

File: pages/main-parent/components/HomePage.jsx (hunk 1 above)
Comment (mohit) [inline pages/main-parent/components/HomePage.jsx]: 🍭

**Comment by mohit** [comment]
cool. going to start working on Google Auth in another branch

**Pull request merged**